cloudbuild/e2e-test-scripts/table_read.sh

```diff
@@ -40,6 +40,8 @@ elif [ "$MODE" == "unbounded" ]
 then
   echo "Unbounded Mode!"
   source cloudbuild/e2e-test-scripts/unbounded_table_read.sh
+  # Add more sleep time. as IO might take time.
+  sleep 55
```

Comment:
Why 55 seconds? Why not a round number like 1 minute?

Reply:
Because you plan to sleep 5 seconds at the end?

Reply:
Its still better to use 60 seconds here. Prefer round numbers for arbitrary usage.